### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by CadenceCase**

### CODE REVIEW

File: src/fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default function fibonacci(n){
```
Comment (nodatall) [inline src/fibonacci.js]: Using descriptive variable names is wonderful! `num` or `number` instead of `n`

File: src/fibonacci.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  if(n === 1){
+    return [0];
+  }else if(n > 1){
```
Comment (nodatall) [inline src/fibonacci.js]: I find `if` and `else if` with no space around them hard to read, I'd prefer:
```js
} else if (n > 1) {
```

File: src/fibonacci.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    var array = [0, 1];
+    var tempArray = [];
+    while(n>2){
+      tempArray = (array.slice(-2));
+      var nextTerm = tempArray.reduce(function(a,b) {return a+b;});
+      array.push(nextTerm);
```
Comment (nodatall) [inline src/fibonacci.js]: Nice use of `splice` and `reduce` to add the last to values together. You could condense it by removing tempArray:
```js
var array = [0, 1];

while (n > 2) { 
  var nextTerm = array.slice(-2).reduce(function(a,b) {return a+b;});
  array.push(nextTerm);
  n--;
}
```

File: src/fibonacci.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+export default function fibonacci(n){
+  if(n === 1){
+    return [0];
+  }else if(n > 1){
+    var array = [0, 1];
+    var tempArray = [];
+    while(n>2){
+      tempArray = (array.slice(-2));
+      var nextTerm = tempArray.reduce(function(a,b) {return a+b;});
```
Comment (nodatall) [inline src/fibonacci.js]: Nice descriptive variable name `nextTerm` !

File: src/fibonacci.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+export default function fibonacci(n){
+  if(n === 1){
+    return [0];
+  }else if(n > 1){
+    var array = [0, 1];
```
Comment (nodatall) [inline src/fibonacci.js]: This is more like `resultsArray` or `fibonacciArray`

File: test/fibonacci_test.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  })
+
+  it('returns fibonacci sequence to nth position', function(){
+    expect(fibonacci(1)).to.deep.equal([0])
+    expect(fibonacci(2)).to.deep.equal([0, 1])
+    expect(fibonacci(10)).to.deep.equal([0, 1, 1, 2, 3, 5, 8, 13, 21, 34])
+    expect(fibonacci(5)).to.deep.equal([0, 1, 1, 2, 3])
+  })
+
+  it('does not work with non-numbers, zero, or negative numbers', function(){
```
Comment (nodatall) [inline test/fibonacci_test.js:16]: Awesome testing edge cases!

File: src/isPalindrome.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(inputString){
+
+  inputString = inputString.toLowerCase();
+  inputString = inputString.replace(/[^\w]|_/g, "");
```
Comment (nodatall) [inline src/isPalindrome.js:4]: Could condense this to one line and assign it to a variable that describes what it is:
```js
var inputOnlyLetters = inputString.toLowerCase().replace(/[^\w]|_/g, "");
```

File: src/isPalindrome.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+  var inputArray = inputString.split('');
+  inputArray.reverse();
+  var reverseString = inputArray.join('');
```
Comment (nodatall) [inline src/isPalindrome.js:7]: Could also be on one line:
```js
var reverseString = inputString.split('').reverse().join('');
```

File: src/isPalindrome.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+  if (reverseString === inputString) {
+    return true;
+  } else {
+    return false;
```
Comment (nodatall) [inline src/isPalindrome.js:11]: I like ternaries: 
```js
return reverseString === inputString ? true : false
```

File: test/isPalindrome_test.js [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+  it('should be a function', function(){
+    expect(isPalindrome).to.be.a('function')
+  })
+
+  it('returns true if inputString is palindrome', function(){
+    expect(isPalindrome('racecar')).to.deep.equal(true)
+    expect(isPalindrome('A man, a plan, a canal: Panama')).to.deep.equal(true)
+  })
+
+  it('returns false if inputString is not palindrome', function(){
```
Comment (nodatall) [inline test/isPalindrome_test.js:15]: Maybe add a test with a string that contains underscores, since your regex was looking for them.
